src/sentry/static/sentry/app/components/eventOrGroupHeader.jsx

```diff
@@ -27,6 +27,7 @@ class EventOrGroupHeader extends React.Component {
         'expectct',
         'expectstaple',
         'default',
+        'transaction',
```

Comment:
This looks good. Should we have a type in SentryTypes for the valid event types? It could save on some duplication.